src/main/resources/swagger.api/schemas/common.yaml

```diff
@@ -0,0 +1,112 @@
+uuid:
+  type: string
+  format: uuid
+
+Metadata:
```

Comment:
I dont think this will be required. We are not  getting as part of the DCB payload request.

Reply:
@gurleenkaurbp :- Metadata class wont be generated in this module, we have kept it as a common schema.

Reply:
got it